### ISSUE TITLE
Missing Variants for Nintendo Switch and Paisley Bandanna

### ISSUE THREAD

**Issue by xeasuperdark** [jefflomacy/villagerdb#151]
![image](https://user-images.githubusercontent.com/64043164/79829482-34498d00-8371-11ea-98f8-ec1a69de0200.png)
![image](https://user-images.githubusercontent.com/64043164/79829496-39a6d780-8371-11ea-8848-a9ec4b987e3a.png)
![image](https://user-images.githubusercontent.com/64043164/79829518-44fa0300-8371-11ea-9c9c-090908d89aa2.png)
![image](https://user-images.githubusercontent.com/64043164/79829523-49beb700-8371-11ea-9ad3-fd511a529998.png)
![image](https://user-images.githubusercontent.com/64043164/79829537-4fb49800-8371-11ea-883b-fb09b747cbd6.png)
![image](https://user-images.githubusercontent.com/64043164/79829547-55aa7900-8371-11ea-9765-6b3163efe1f4.png)
![image](https://user-images.githubusercontent.com/64043164/79829559-5b07c380-8371-11ea-957b-5eb2851bdeec.png)
![image](https://user-images.githubusercontent.com/64043164/79829564-5fcc7780-8371-11ea-9074-a3c4d0ad693b.png)
![image](https://user-images.githubusercontent.com/64043164/79829572-6529c200-8371-11ea-9144-8371b6578978.png)

**Comment by jefflomacy** [comment]
Agreed on the switch. For the Bandanna, though, I see 8 variations. Which one are we missing, if any? 
https://villagerdb.com/item/paisley-bandanna

**Comment by xeasuperdark** [comment]
Seems I as mistaken on Paisley-bandanna. My bad, thanks and keep up the good work!    On Wednesday, April 22, 2020, 08:28:20 AM EDT, jefflomacy <notifications@github.com> wrote:  
 
 


Agreed on the switch. For the Bandanna, though, I see 8 variations. Which one are we missing, if any?
https://villagerdb.com/item/paisley-bandanna

**Comment by jefflomacy** [comment]
Thanks! We will get the switch updated and close this ticket once we verify it. You're always welcome to re-open a ticket if we miss something.
